### ISSUE TITLE
Layout problem in 11.8 OCB mode

### ISSUE THREAD

**Issue by postboy** [crypto101/book#284]
Pages 140-141 look like this:

![image](https://cloud.githubusercontent.com/assets/12019610/16585954/0c2937f6-42cc-11e6-8010-cd1a19547336.png)

So, there's a problem:
*text here* That associated data tag ta is computed as follows:
*and then again goes text on the other topic*
*and corresponding picture is only on the other page, not on current*

So, I think that this picture should be referenced as "That associated data tag ta is computed as shown on figure 123" or text "This design has a number of interesting properties..." should start only after the image!

**Comment by lvh** [comment]
Yeah, I should do that with an explicit figure references. Making images show up at a very particular spot is unfortunately tricky in LaTeX (arguably because you shouldn't want that).

**Comment by postboy** [comment]
Found same problem on another page:

![image](https://cloud.githubusercontent.com/assets/12019610/16619115/6e6c1626-4395-11e6-904a-ef4754dd77a1.png)
